scripts/apps/desks/controllers/DeskConfigController.ts

```diff
@@ -1,24 +1,31 @@
 import _ from 'lodash';
 import {gettext} from 'core/utils';
 
-DeskConfigController.$inject = ['$scope', '$controller', 'notify', 'desks', 'WizardHandler', 'modal'];
-export function DeskConfigController($scope, $controller, notify, desks, WizardHandler, modal) {
+DeskConfigController.$inject = ['$scope', '$controller', 'notify', 'desks', 'modal', '$rootScope'];
+export function DeskConfigController($scope, $controller, notify, desks, modal, $rootScope) {
     // expecting $scope.desks to be defined
 
+    const stepsToWatch = ['general', 'people'];
```

Comment:
Can you check if data changed, no matter which tab is opened?

Reply:
We can, but making changes in the other two tabs do not affect the desk. That's why avoided watching all the tabs.

Reply:
Do that then. I'd rather have less code which is easier to maintain.